.github/workflows/android.yml

```diff
@@ -34,13 +34,15 @@ jobs:
       - name: Install npm dependencies
         run: |
           npm ci
+      - name: Create .env.local file
+        run: |
+          echo "${{ secrets.ENV_FILE }}" > .env.local
```

Comment:
@pmigueld I just want to confirm if do we need to add creation of `android/local.properties` here in the yml? or will the .env be enough?

Reply:
oh right, we need that too for Android. I'll add it.